statistics/table.go

```diff
@@ -239,3 +258,97 @@ func PseudoTable(ti *model.TableInfo) *Table {
 	}
 	return t
 }
+
+func getPseudoRowCountByIndexRanges(sc *variable.StatementContext, indexRanges []*types.IndexRange, inAndEQCnt int,
```

Comment:
The current algorithm is not good, will be changed later.
We can add test cases when we change it.